source/common/rds/route_config_update_receiver_impl.cc

```diff
@@ -8,14 +8,15 @@ namespace Rds {
 RouteConfigUpdateReceiverImpl::RouteConfigUpdateReceiverImpl(
     ConfigTraits& config_traits, ProtoTraits& proto_traits,
     Server::Configuration::ServerFactoryContext& factory_context)
-    : config_traits_(config_traits), proto_traits_(proto_traits),
+    : config_traits_(config_traits), proto_traits_(proto_traits), factory_context_(factory_context),
       time_source_(factory_context.timeSource()),
       route_config_proto_(proto_traits_.createEmptyProto()), last_config_hash_(0ull),
       config_(config_traits_.createNullConfig()) {}
 
 void RouteConfigUpdateReceiverImpl::updateConfig(
     std::unique_ptr<Protobuf::Message>&& route_config_proto) {
-  config_ = config_traits_.createConfig(*route_config_proto);
+  config_ = config_traits_.createConfig(*route_config_proto, factory_context_,
+                                        false /* not validate unknown cluster */);
```

Comment:
There is a `RouteConfiguration` option to do this validation IIRC,  do you have the same in TRDS? Or are you just relying on always validating?

Reply:
HTTP has an option to change.
https://github.com/envoyproxy/envoy/blob/a308789897793b111ee65715c5ddb4a58b775083/api/envoy/config/route/v3/route.proto#L110

I don't have the same in TRDS at the moment. It should be coming soon in this diff or the upcoming diff.

Reply:
FWIW I'm talking about https://github.com/envoyproxy/envoy/blob/b34fcee23881be2e9d5b6e5f04ca6cba02ef4539/api/envoy/config/route/v3/route.proto#L97

Reply:
OK, I see. Can the PR title/description/comments be reflected to show this state? I feel it's a bit confusing trying to understand the dynamics here between static vs. dydnamic validation in RDS vs. TRDS.

Reply:
Done. Thanks